lib/perl/Genome/Model/Tools/DetectVariants2/Dispatcher.pm

```diff
@@ -102,6 +102,9 @@ class Genome::Model::Tools::DetectVariants2::Dispatcher {
         lsf_queue => {
             default_value => Genome::Config::get('lsf_queue_dv2_workflow'),
         },
+        lsf_resource => {
+            default => Genome::Config::get('lsf_resource_dv2_dispatcher'),
```

Comment:
These are synonyms: default, value and default_value. The default and value both normalize to default_value.